Gems/Atom/Feature/Common/Assets/Materials/Types/BasePBR_Vertex.azsli

```diff
@@ -0,0 +1,56 @@
+/*
+ * Copyright (c) Contributors to the Open 3D Engine Project.
+ * For complete copyright and license terms please see the LICENSE at the root of this distribution.
+ *
+ * SPDX-License-Identifier: Apache-2.0 OR MIT
+ *
+ */
+
+#pragma once
+
+
+#include <Atom/RPI/TangentSpace.azsli>
+
+VSOutput EvaluateVertexGeometry_BasePBR(
```

Comment:
super nit: Evaluate is a word that implies a different meaning. Perhaps 'Generate' or 'Fill' will be better suited here. For example 'FillVertexGeometryData..'